ovirt/resource_ovirt_disk_from_image_test.go

```diff
@@ -0,0 +1,54 @@
+package ovirt
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestImageUpload(t *testing.T) {
+	t.Parallel()
+
+	p := newProvider(newTestLogger(t))
+	storageDomainID := p.getTestHelper().GetStorageDomainID()
+
+	resource.UnitTest(
+		t, resource.TestCase{
+			ProviderFactories: p.getProviderFactories(),
+			Steps: []resource.TestStep{
+				{
+					Config: fmt.Sprintf(
+						`
+provider "ovirt" {
+	mock = true
+}
+
+resource "ovirt_disk_from_image" "foo" {
+	storagedomain_id = "%s"
+	format           = "raw"
+    alias            = "test"
+    sparse           = true
+    source_file      = "./testimage/image"
+}
+`,
+						storageDomainID,
+					),
+					Check: resource.ComposeTestCheckFunc(
+						resource.TestMatchResourceAttr(
+							"ovirt_disk_from_image.foo",
+							"storagedomain_id",
+							regexp.MustCompile(fmt.Sprintf("^%s$", regexp.QuoteMeta(string(storageDomainID)))),
+						),
+						resource.TestMatchResourceAttr(
+							"ovirt_disk_from_image.foo",
+							"size",
+							regexp.MustCompile(fmt.Sprintf("^%d$", 1024*1024)),
```

Comment:
The `1024*1024` is the minimal disk size, right? Maybe we could add a constant for it? (either here or in go-ovirt-client)

Reply:
Technically no, 4kB is the minimal disk size, but that causes the oVirt Engine to crash. We implemented a safe minimum of 1 MB.

Reply:
Ah, true. A constant would be great in my point of view, otherwise it might seem a magic number. But not in this PR, since this would require changes in go-ovirt-client.